app/src/main/java/com/rob729/quiethours/Activity/SplashScreen.kt

```diff
@@ -12,6 +15,13 @@ class SplashScreen : AppCompatActivity() {
         val intent = Intent(this, MainActivity::class.java)
         startActivity(intent)
         overridePendingTransition(R.anim.fade_in, R.anim.fade_out)
+        checkFirstRun()
         finish()
     }
+    fun checkFirstRun() {
+        if (!StoreSession.getNightMode(AppConstants.FIRST_BOOT)) {
```

Comment:
what is the logic of getting AppConstants.FIRST_BOOT from StoreSession.getNightMode() ?